app/src/Components/Project/ProjectSkills.tsx

```diff
@@ -25,7 +25,7 @@ const ProjectSkills: FC<ProjectSkillsProps> = ({ skills }) => {
     <>
       {skills.map(skill => (
         <SkillsView key={skill}>
-          <Entypo name="tools" size={16} />
+          <Entypo name="person-circle-outline" size={16} />
```

Comment:
I think this was changed while someone was testing out different icons -- was it a deliberate change you want to include in the PR or should it be left unaltered?